docs/diagrams/wallet-lifecycle/wallet-lifecycle-1-handling-frauds.tikz

```diff
@@ -0,0 +1,187 @@
+\node[revert, below right=of fc_check1]                          (fc_revert1) {Transaction reverted};
+\node[check, below left=of fc_check1, xshift=-1cm]               (fc_check2)  {Is wallet LIVE,\\ MOVING\_FUNDS\\ or CLOSING?};
+\node[check, below right=of fc_check2]                           (fc_check3)  {Has challenger deposited\\ required amount?};
+\node[revert, below left=of fc_check3, xshift=-3cm]              (fc_revert2) {Transaction reverted};
+\node[action, below right=of fc_check3]                          (fc_action1) {Save time (starting point for defeat timeout calculation)};
+\node[note1, right=of fc_start.south east, anchor=south west]    (fc_note1)   {Can be successfully called for wallets:\\
+                                                                               \textbf{- LIVE\\
+                                                                               - MOVING\_FUNDS\\
+                                                                               - CLOSING}};
+\node[note2, above=of fc_note1.north, anchor=south, yshift=-8mm] (fc_note2)   {Sender incentivized by:\\ \textbf{Fraction of slashed amount if fraud doesn't get defeated}};
+
+% Arrows
+\draw[->] (fc_start)   -- (fc_check1);
+\draw[->] (fc_check1)  -| (fc_check2)  \no;
+\draw[->] (fc_check1)  -| (fc_revert1) \yes;
+\draw[->] (fc_check2)  -| (fc_revert2) \no;
+\draw[->] (fc_check2)  -| (fc_check3)  \yes;
+\draw[->] (fc_check3)  -| (fc_revert2) \no;
+\draw[->] (fc_check3)  -| (fc_action1) \yes;
+
+
+% DEFEAT FRAUD
+% Nodes
+\node[start, right=of fc_start, xshift=100mm]                    (df_start)   {\textbf{Defeat fraud challenge}};
+\node[check, below=of df_start]                                  (df_check1)  {Is there a pending\\ notification to be defeated?};
+\node[check, below right=of df_check1]                           (df_check2)  {Has the defeat timeout\\ been reported?};
+\node[check, below left=of df_check2]                            (df_check3)  {Were the UTXOs\\ in the challenged transaction\\ spent non-fraudulently?};
+\node[revert, below right=of df_check2]                          (df_revert1) {Transaction reverted};
+\node[revert, below left=of df_check3]                           (df_revert2) {Transaction reverted};
+\node[action, below right=of df_check3, yshift=-3cm]             (df_action1) {Close the fraud challenge, don't allow for challenging again)};
+\node[action, below=of df_action1]                               (df_action2) {Transfer the challenger's deposit to the treasury};
+\node[note1, right=of df_start.south east, anchor=south west]    (df_note1)   {Can be successfully called for wallets:\\ 
+                                                                               \textbf{- LIVE\\
+                                                                               - MOVING\_FUNDS\\
+                                                                               - CLOSING}};
```

Comment:
*Defeat fraud challenge*, *Defeat fraud challenge with timeout*, and *Send notification about fraud defeat timeout* can also be called for Terminated wallets.

We need to allow notification about the fraud defeat timeout so that the challenger can have their ETH back in case the wallet was terminated for other reasons in the meantime.

Given we allow the challenger to send this notification for Terminated wallet, it also makes sense to allow the Terminated wallet to defeat the challenge. Realistically, Terminated wallet might not be interested in defeating the challenge but giving the wallet a chance makes sense, at least for consistency.